src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenReadOnlySpanConstructionTest.cs

```diff
@@ -1586,7 +1586,7 @@ .maxstack  2
 ");
         }
 
-        [Fact]
+        [ConditionalFact(typeof(IsEnglishLocal))]
```

Comment:
Failed in Spanish VM
![image](https://github.com/dotnet/roslyn/assets/24360909/93a67d45-a2c6-4b4f-81e7-1d6420344e1a)
So I changed this to English only

Reply:
Talked to Compiler folks offline and verifed this is OK

Reply:
@Cosifne

> Talked to Compiler folks offline and verifed this is OK

I think we should open an issue for the test infrastructure. It should make sure that verification errors are obtained in English. Then this change should be reverted

Reply:
Opened #71509.